llama_stack/apis/post_training/post_training.py

```diff
@@ -182,7 +182,19 @@ async def supervised_fine_tune(
         ),
         checkpoint_dir: str | None = None,
         algorithm_config: AlgorithmConfig | None = None,
-    ) -> PostTrainingJob: ...
+    ) -> PostTrainingJob:
+        """Supervised fine-tune a model
+
+        :param job_uuid: The UUID of the job to create.
+        :param training_config: The training configuration.
+        :param hyperparam_search_config: The hyperparam search configuration.
+        :param logger_config: The logger configuration.
+        :param model: The model to fine-tune.
+        :param checkpoint_dir: The directory to save the checkpoint to.
```

Comment:
I think it's actually "save checkpoints" since there may be more than one? (This argument should be removed eventually...)